maxentep/maxent.py

```diff
@@ -28,13 +28,13 @@ def traj_to_restraints(traj, inner_slice, npoints, prior, noise=0.1, time_averag
 
         def fxn(x, s=s, j=inner_slice):
             return tf.reduce_mean(x[s], axis=0)[j]
-        print(i * time_average + time_average // 2,
+        print(i * time_average + time_average // 2 + start_time,
```

Comment:
Probably not needed

Reply:
I made it so the user can define a time period for the observables to be selected from.